recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set rev = "d479f7a9" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "d479f7a9" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  script: CFLAGS='-I${PREFIX}/include/python${CONDA_PY}m' python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - anaconda
+    - cython
+    - sympy
+    - pytables
+    - mpi4py
+    - scikit-umfpack
+
+  run:
+    - python
+    - anaconda
+    - pyparsing
+    - sympy
+    - cython
+    - pytables
+    - mpi4py
+    - scikit-umfpack
+
+about:
+  home: http://{{ name }}.org
```

Comment:
Please just use the full explicit url so that it's easy to copy-and-paste if someone is inspecting the recipe and wants to go to the site.